arlas-rest/src/main/java/io/arlas/server/rest/explore/search/SearchRESTService.java

```diff
@@ -59,121 +64,127 @@ public SearchRESTService(ExploreService exploreService) {
+            @Parameter(name = "q",
+                    description = Documentation.FILTER_PARAM_Q,
+                    style = ParameterStyle.FORM,
+                    explode = Explode.TRUE)
             @QueryParam(value = "q") List<String> q,
 
-            @ApiParam(name = "dateformat",
-                    value = Documentation.FILTER_DATE_FORMAT)
+            @Parameter(name = "dateformat",
+                    description = Documentation.FILTER_DATE_FORMAT)
             @QueryParam(value = "dateformat") String dateformat,
 
-            @ApiParam(name = "righthand",
-                    defaultValue = "true",
-                    value = Documentation.FILTER_RIGHT_HAND)
+            @Parameter(name = "righthand",
+                    schema = @Schema(defaultValue = "true"),
+                    description = Documentation.FILTER_RIGHT_HAND)
             @QueryParam(value = "righthand") Boolean righthand,
 
-            @ApiParam(hidden = true)
+            @Parameter(hidden = true)
             @HeaderParam(value = PARTITION_FILTER) String partitionFilter,
 
-            @ApiParam(hidden = true)
+            @Parameter(hidden = true)
             @HeaderParam(value = COLUMN_FILTER) String columnFilter,
 
-            @ApiParam(hidden = true)
+            @Parameter(hidden = true)
             @HeaderParam(value = ARLAS_ORGANISATION) String organisations,
 
             // --------------------------------------------------------
             // -----------------------  FORM    -----------------------
             // --------------------------------------------------------
-            @ApiParam(name = "pretty",
-                    value = Documentation.FORM_PRETTY,
-                    defaultValue = "false")
+            @Parameter(name = "pretty",
+                    description = Documentation.FORM_PRETTY,
+                    schema = @Schema(defaultValue = "false"))
             @DefaultValue("false")
             @QueryParam(value = "pretty") Boolean pretty,
 
-            @ApiParam(name = "flat",
-                    value = Documentation.FORM_FLAT,
-                    defaultValue = "false")
+            @Parameter(name = "flat",
+                    description = Documentation.FORM_FLAT,
+                    schema = @Schema(defaultValue = "false"))
             @QueryParam(value = "flat") Boolean flat,
 
             // --------------------------------------------------------
             // -----------------------  PROJECTION   -----------------------
             // --------------------------------------------------------
 
-            @ApiParam(name = "include",
-                    value = Documentation.PROJECTION_PARAM_INCLUDE,
-                    allowMultiple = true,
-                    defaultValue = "*")
+            @Parameter(name = "include",
+                    description = Documentation.PROJECTION_PARAM_INCLUDE,
+                    style = ParameterStyle.FORM,
+                    explode = Explode.TRUE,
```

Comment:
if we put explode to true, I think we then need to accept a List<string> include, i think we should not explode even though we used to say before that :allowMultiple = true => it was a mistake I'd say

Reply:
Done